keps/sig-multicluster/2149-clusterid/README.md

```diff
@@ -307,7 +317,7 @@ The schema for `ClusterProperty` is intentionally loose to support multiple form
 
 ### Well known properties
 
-The `ClusterProperty` CRD will support two specific properties under the well known names `id.k8s.io` and `clusterset.k8s.io`. Being "well known" means that they must conform to the requirements described below, and therefore can be depended on by multi-cluster implementations to achieve use cases dependent on knowledge of a cluster's ID or ClusterSet membership.
+The `ClusterProperty` CRD will support three specific properties under the well known names `id.k8s.io`, `network.k8s.io` and `clusterset.k8s.io`. Being "well known" means that they must conform to the requirements described below, and therefore can be depended on by multi-cluster implementations to achieve use cases dependent on knowledge of a cluster's ID or ClusterSet membership.
```

Comment:
I asked somewhere (can't find where) about whether we can/should revisit the need for suffix for "well known" properties vs user-defined.  I.e could/should these be "id", "network", "clusterset" ?